src/ripple/app/tx/impl/SetSignerList.h

```diff
@@ -85,8 +91,11 @@ class SetSignerList : public Transactor
     TER replaceSignerList ();
     TER destroySignerList ();
 
-    TER removeSignersFromLedger (Keylet const& accountKeylet,
-        Keylet const& ownerDirKeylet, Keylet const& signerListKeylet);
+    static
+    TER removeSignersFromLedger (
```

Comment:
Why a private static? I'd expect this to be a file-scoped function in the `cpp`. Because of unity build issues or something else?